Tools/Scripts/libraries/webkitscmpy/webkitscmpy/mocks/local/git.py

```diff
@@ -801,52 +814,6 @@ def filter_branch(self, range, identifier_template=None, environment_shell=None,
             stdout=stdout.getvalue(),
         )
 
-    def commits_in_range(self, begin, end):
```

Comment:
Little concerned to see this function deleted? Can you share more your thinking?

Reply:
This is removed and replaced by rev_list, rev_list implement two dots and triple dots syntax following the git documentation. So for A..B means A u B - A, and for A...B means A u B - A n B, there actually no begin or end concept.